src/components/numbers/index.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { FC } from 'react'
+
+import { NUMBERS } from 'typings'

```

Comment:
I noticed that sometimes you import your types form 'typings' and sometimes you do it from '../../typings'. I understand that based on the directory you're currently in, the path to the typings file is different. However, even when the typings file is in the same level and you can just do 'typings', it is a good practice to always add the path (`'./typings'` instead of just `'typings'`). This is because not having a path makes it look like you are working with an npm package, so adding the path is always good to make it clear that it is a file.

In the future, I will teach you how to create aliases, so you can make reference to imports by using aliases. so for this, if the typings is a file you're constantly including, you could create an alias named `@typings` and always do `import { MY_TYPE } from '@typings'`. I will show you how this works on the design system project ;)